schema/artwork/context.js

```diff
@@ -39,9 +39,9 @@ export default {
   resolve: ({ id }) =>
     Promise
       .all([
-        gravity('related/fairs', { artwork: [id], active: true, size: 1 }),
-        gravity('related/sales', { artwork: [id], active: true, size: 1 }),
-        gravity('related/shows', { artwork: [id], active: true, size: 1 }),
+        gravity('related/fairs', { artwork: [id], size: 1 }),
+        gravity('related/sales', { artwork: [id], size: 1 }),
```

Comment:
Actually no; believe it or not there are still normal sales. https://www.artsy.net/artwork/andy-warhol-electric-chair-18